starboard/android/shared/player_components_factory.h

```diff
@@ -406,14 +414,18 @@ class PlayerComponentsFactory : public starboard::shared::starboard::player::
       SB_DCHECK(video_render_algorithm);
       SB_DCHECK(video_renderer_sink);
       SB_DCHECK(error_message);
+      // Set max_input_size with a non-zero value to overwrite
+      // MediaFormat.KEY_MAX_INPUT_SIZE
+      int max_input_size = video_mime_type.GetParamIntValue("maxinputsize", 0);
```

Comment:
nit: would -1 also work instead of 0? It's a minor detail, but 0 could read that we're setting the max input size to 0 rather than leaving it as default. I think -1 can convey the idea better, like how `tunnel_mode_audio_session_id` is -1 for non tunnel-mode playback.

Reply:
Fixed.